cas-server-support-osf/src/main/java/io/cos/cas/adaptors/mongodb/OpenScienceFrameworkLogoutHandler.java

```diff
@@ -1,118 +0,0 @@
-/*
```

Comment:
TO CR: this file is renamed to `OpenScienceFrameworkInstitutionAuthenticationHandler`. The one commit for the rename consider it `mv` but the PR does not. There are a two main changes: 1. add all institution related fields to `OpenScienceFrameworkInstitution` 2. add a method to return a map of institution name and login url.